pages/dkp/konvoy/2.2/Install/index.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: layout.pug
+navigationTitle: Install
+title: Install
+menuWeight: 80
+excerpt: Install a Konvoy cluster
+beta: false
+enterprise: false
+---
+
+When installing DKP for a project, line-of-business, or enterprise, the first step is to determine the infrastructure on which you want to deploy.
+
+For example, you can:
+
+- Install on a public cloud infrastructure, such as Amazon Web Services (AWS) or Azure.
+- Install on an internal network on-premises with a physical (bare metal) or virtual infrastructure.
+- Install on air-gapped Edge.
+
+<p class="message--important"><strong>IMPORTANT: </strong>The infrastructure you select then determines the specific requirements for a successful installation.</p>
+
+
+Konvoy is a tool for provisioning Kubernetes clusters with a suite of pre-selected [Cloud Native Computing Foundation (CNCF)][cncf] and community-contributed tools.
+By combining a native Kubernetes cluster as its foundation with a default set of cluster extensions,
+Konvoy provides a complete out-of-the-box solution for organizations that want to deploy production-ready
+Kubernetes.
+
+This Quick Start guide provides simplified instructions to get your Konvoy cluster up and running with minimal configuration requirements on an Amazon Web Services (AWS) public cloud instance.
```

Comment:
I think we should take out the AWS reference as we want to support other providers? Or if this is AWS-specific, surface that in the title?

Reply:
I think these are part of 1.8 instructions and I should probably modify this to say the example is AWS specific.  I know I included the "Choose your infrastructure" link after initial installation to further the instructions, but honestly, you're right, this should be worded different.